ui/perfherder/perf-helpers/constants.js

```diff
@@ -12,6 +12,7 @@ export const tValueConfidence = 5; // Anything above this is "high" in confidenc
 export const endpoints = {
   alert: '/performance/alert/',
   alertSummary: '/performance/alertsummary/',
+  alertSummarySimple: '/performance/alertsummarysimple/',
```

Comment:
If I'm not mistaken this endpoint is no longer needed since you approached a different solution.